.gitignore

```diff
@@ -1,5 +1,17 @@
-# Only project- and language-specific ignores in here. Use global .gitignore for editors etc
+*.class
+*.log
+
+# sbt specific
+dist/*
+target/
+lib_managed/
+src_managed/
+project/boot/
+project/plugins/project/
 
 # Vagrant
-.vagrant/
+.vagrant
+VERSION
 
+# Intellij
+.idea
```

Comment:
Only project- and language-specific ignores in here. Use global .gitignore for editors etc https://help.github.com/articles/ignoring-files/#create-a-global-gitignore